.envrc

```diff
@@ -1,2 +1,2 @@
 export DATABASE_URL=postgresql://prisma:prisma@localhost:6432/test
-export REPLICA_URL=postgresql://prisma:prisma@localhost:7432/test
\ No newline at end of file
+export REPLICA_URL=postgresql://prisma:prisma@localhost:6432/test
```

Comment:
Now both primary and replica URLs point to the same instance. What was the point of this change? I don't think we can quite trust the tests/demo if actual replication is not used.

Reply:
Oh whoops, that's a dev environment simplification I used. I'll revert that.